PR_Review_And_Contribute_Practices.txt

```diff
@@ -0,0 +1,61 @@
+For reviewers rules:
+============================================================================
+
+1)Everyone who has experiences in the project is encouraged to review PRs.
+2)Respectful, kind, patient to the coders
+3)Freely deny for changes the codebase does not want/need even though perfect design/codes
+4)Ask questions rather than make statements.
+5)Not encourage the “Why” questions.
+    Good practice: e.g. Wouldn't it make more sense to
+    Would you like to? Could you give the reason that ...?
+6)Remember to praise.
+7)Remember that there is often more than one way to approach a solution.
+8)Given clear and useful comments, and explain the reason why reqest change
+9)In general, reviewers should favor approving a PR once it is in a state where it definitely improves the overall code health of the system being worked on,
+  even if the PR isn't perfect.(maintainability, readability, and understandability)
+10)Share your best practice/knowledge as a mentor.
+11)Cautiously regard personal preference as best practice and impose to contributors
+12)Dismiss your approval if you add new comment for a PR after you already have given an approval
+13)It is ok to use a 'request review' tool to ask someone to review as you like, but not a must.
+14)It is ok to cancel the request review to you if you think you are not a suitable one for this PR.
+15)Wait for request review if there is for 2 weeks for those PRs which have already 2 approvals
```

Comment:
I think the description for No.15 is not so clear to understand

Reply:
updated with :
Wait for request review for no more than 2 weeks on those PRs which have already 2 approvals